resolver.go

```diff
@@ -49,16 +50,17 @@ func exprText(x ast.Expr) string {
 			// TODO: handle accents
 			return t.Value
 		default:
-			panic("unhandled ast.Text value: " + t.Value)
+			return t.Value
 		}
 	case *ast.ConcatExpr:
 		left := exprText(t.X)
 		right := exprText(t.Y)
 		return left + right
 	default:
-		panic("unhandled ast.Expr value")
+		return ""
```

Comment:
I'd prefer to panic here and the previous spot. These switches should be exhaustive so not handling a case is a code error that should be fixed.